### ISSUE TITLE
Are there plans to support loading external configuration via Spring's `spring.config.import` mechanism?

### ISSUE THREAD

**Issue by tylervangorder-8451** [Azure/AppConfiguration#887]
Many of the spring cloud projects (spring-config-server, spring-cloud-vault) have moved away from using the `bootstrap.properties/boostrap.yml` to load external configuration early in the Spring application's lifecycle in favor of using the configuration imports.

Are there plans to do something similar with Azure App Configuration? 

For reference:

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-Config-Data-Migration-Guide
https://spring.io/blog/2020/08/14/config-file-processing-in-spring-boot-2-4

**Comment by mrm9084** [comment]
Yes, but it will be a breaking change to support it. We are waiting on the next major release.

**Comment by mrm9084** [comment]
@sadrishya, we actually just did a 6.0.0-beta.1 release with this change. We are currently working on the documentation and the full release.

**Comment by mrm9084** [comment]
The 6.0.0 release that adds support for `spring.config.import` is out.
